### ISSUE TITLE
VB2 on RNA shortgun sequecing

### ISSUE THREAD

**Issue by manifestoso** [Griffan/VerifyBamID#47]
Hi @Griffan, 

Just wondering whether there was precedent to use VB2 on RNA samples, or you can perceive any potential roadblocks?

**Comment by Griffan** [comment]
@manifestoso 
No, we haven't done any experiments on RNA samples using VB2. But we tried to use a closely related mixture model on RNA samples to estimate the kinship, of which the performance is not always stable due to limited dataset back then (very short reads, uneven coverage, etc.).
In VB2, two main roadblocks are: 1) limited germline common variants that overlaps with specific exons/transcripts 2) allele specific expression that leads to imbalanced allele count.
